packages/editor-ui/src/Interface.ts

```diff
@@ -1090,6 +1090,9 @@ export interface IModalState {
 	activeId?: string | null;
 	curlCommand?: string;
 	httpNodeParameters?: string;
+}
+
+export interface NewCredentialsModal extends IModalState {
 	requiredCredentials?: boolean;
```

Comment:
nitpick - still `requiredCredential` is not a great name here..

Reply:
Alright, updated.